### DISCUSSION TITLE
Task 7: Problem 6

### DISCUSSION THREAD

**Discussion by noreengao0924** [chulminy/CIVE497-CIVE700#173] [Q&A]
Hi When I tried to compute the accuracy of F1 using 8 point algorithm, my error would be big like around 1-100 if I added the perturbation:
``
x_img1_list[0, id[0]] += 10
x_img1_list[1, id[0]] += 10
``
If I remove the perturbation, my error is okay.

**Comment by chulminy** [comment]
Why are you adding perturbation?
